mne/channels/channels.py

```diff
@@ -1732,6 +1732,7 @@ def combine_channels(inst, groups, method='mean', keep_stim=False,
     elif isinstance(inst, BaseEpochs):
         combined_inst = EpochsArray(new_data, info, tmin=inst.times[0],
                                     verbose=inst.verbose)
+        combined_inst.events = inst.events
```

Comment:
the `EpochsArray` constructor has an `events` parameter. Does it work to pass `events=inst.events` to the constructor? seems simpler.

Reply:
Perfect, thanks!